webpack.dev.js

```diff
@@ -19,6 +19,7 @@ katexConfig.devServer = {
     disableHostCheck: true,
     host: '0.0.0.0',
     port: PORT,
+    sockPort: 'location',
```

Comment:
The docs do say anything about `"location"`.  What does this setting do?

Reply:
@kevinbarabash It makes `webpack-dev-server` to use the browser's location port as the socket port: https://github.com/webpack/webpack-dev-server/pull/2341. For instance, when developing locally in `http://localhost:7936`, it uses `7936`, and using GitPod or codespaces, they are tunneled using HTTPS, so it uses `443`. This eliminates the need to set `--sockPort 443` manually.